js/blockchaincom.js

```diff
@@ -229,33 +243,54 @@ module.exports = class blockchaincom extends Exchange {
             } else {
                 maxOrderSize = undefined;
             }
-            const limits = {
-                'amount': {
-                    'min': minOrderSize,
-                    'max': maxOrderSize,
-                },
-                'price': {
-                    'min': undefined,
-                    'max': undefined,
-                },
-                'cost': {
-                    'min': undefined,
-                    'max': undefined,
-                },
-            };
-            const symbol = base + '/' + quote;
             result.push ({
                 'id': marketId,
                 'numericId': numericId,
-                'symbol': symbol,
+                'symbol': base + '/' + quote,
                 'base': base,
                 'quote': quote,
+                'settle': undefined,
                 'baseId': baseId,
                 'quoteId': quoteId,
-                'precision': precision,
-                'limits': limits,
+                'settleId': undefined,
+                'type': 'spot',
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
                 'active': active,
-                'info': market,
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'precision': {
+                    'price': pricePrecision,
+                    'amount': amountPrecision,
+                },
+                'limits': {
+                    'leverage': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': minOrderSize,
+                        'max': maxOrderSize,
+                    },
+                    'price': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'cost': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                },
+
```

Comment:
We cannot have empty lines inside methods, let's always build and test: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-build--run-tests-on-your-local-machine

Reply:
Is there any reason for me NOT to just push it and let Travis and continuous-integration/appveyor/pr do the testing?

It takes about 5-10 minutes to test, and I can't switch branches while that's happening, that adds up, I pushed almost 10 branches yesterday

Reply:
I mean a simple syntax check prior to pushing would save your time in this case, but it's up to you. Sure we can use Travis for that.

Reply:
I have a live syntax checker, but it's not working for blank lines for some reason

Reply:
How can I view the build? I want to check what rule was broken